atomic

```diff
@@ -273,6 +280,9 @@ if __name__ == '__main__':
     imagesp.set_defaults(func='images')
     imagesp.add_argument("--prune", action="store_true",
                          help=_("prune dangling images"))
+    imagesp.add_argument("--oci", dest="oci",
+                         action='store_true', default=False,
+                         help=_('Show OCI containers'))
 
```

Comment:
there is no rootfs assigned yet for the `--oci` flag.  These are the images available in the OSTree repository, which could have or not been checked-out.  The checkout from the OSTree repository is done by `install --oci`.  There could be different containers using the same image.